### ISSUE TITLE
pa11y audits don't provide issue details

### ISSUE THREAD

**Issue by benjamindavid** [mfrachet/cypress-audit#33]
I'm running pa11y on a page that contains an issue, but issue reported doesn't contain any details:
```
cy.pa11y - 1 accessibility violation was found

Issue: WCAG2A.Principle3.Guideline3_2.3_2_2.H32.2, # of occurrences: 1.
- undefined
- undefined
```

It [looks like](https://github.com/mfrachet/cypress-audit/blob/master/src/a11y/commands.js#L43) it's trying to output `issue.description` and `issue.link` when pa11y actually provides `issue.context`, `issue.message` and `issue.selector`:
https://github.com/pa11y/pa11y#javascript-interface

**Comment by mfrachet** [comment]
It should be fixed in #35 . I'm releasing a new version in the day 😊

**Comment by mfrachet** [comment]
Available in 0.2.3 😄
